docs/engine.md

```diff
@@ -19,14 +32,29 @@ Currently, the engines that are supported by DJL are:
 - [PaddlePaddle](../engines/paddlepaddle/README.md) - supports basic inference
 - [TFLite](../engines/tflite/tflite-engine/README.md) - supports basic inference
 - [TensorRT](../engines/tensorrt/README.md) - supports basic inference
+- [XGBoost](../engines/ml/xgboost/README.md) - supports basic inference
+- [LightGBM](../engines/ml/lightgbm/README.md) - supports basic inference
 - [DLR](../engines/dlr/README.md) - supports basic inference
+- [Python](https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/README.md) - supports basic inference
```

Comment:
I find it a bit confusing to include Python engine here since it's provided primarily for the model server use-case and I don't think users will typically be using it with core DJL. 

Since it is an engine, I understand listing alongside the others here. But, it feels like it's different enough than the other basic inference engines that we may want to include a bit more description here, or omit it from this doc?

Reply:
Will remove this